src/serializers/plain.js

```diff
@@ -38,6 +38,18 @@ function deserialize(string, options = {}) {
   return options.toRaw ? raw : Raw.deserialize(raw)
 }
 
+function getSerializedTextForNode(node) {
```

Comment:
Can you rename this to `serializeNode`, and can you move it below `serialize` and give it a DocBlock-style comment to match the others?

Reply:
Done! (And sorry about that missing the comment block!)